### PR TITLE
doc: Rewrite shell example to not leave secrets in the history file

### PR DESCRIPTION
**Pull request by paveljanik**
Variant to #8436.

### CODE REVIEW

**Comment by paveljanik** [comment]
... will squash in a minute ...

File: doc/release-notes.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ For example:
 
     $ echo -e "mysecretcode\n120" | src/bitcoin-cli -stdin walletpassphrase
 
+Warning: when used as described, the command line (including your secret
+walletpassphrase) will be stored in your shell history file. Modify your
+command line or your shell settings to prevent it.
+
 It is recommended to use this for sensitive information such as wallet
```
Comment (laanwj) [inline doc/release-notes.md]: I don't think this quite connects with the next paragraph anymore..

**Comment by laanwj** [comment]
I think this warning should be separate, at the end of the section. It might be worthwhile to add it, as a reminder, but commands possibly ending up in shell history is nothing new. It has nothing to do with this functionality change.

**Comment by paveljanik** [comment]
What about dropping echo instead?

**Comment by laanwj** [comment]
That's fine with me. Make sure that the Ctrl-D stands out as a key action, not literally typing Ctrl-D.

**Comment by paveljanik** [comment]
Done. I'm open for better "stand out" for Ctrl-D :-)

**Comment by sipa** [comment]
utACK 719208c66f2df124cdf542ae80c2f99563fa7851

**Comment by MarcoFalke** [comment]
utACK 719208c

**Pull request merged**